sicario.c

```diff
@@ -214,3 +323,101 @@ void ncursesInit() {
 void ncursesEnd() {
 	endwin();
 }
+
+void createPlayers() {
+	playerCount = PLAYER_AMOUNT;
+	for (int i = 0; i < PLAYER_AMOUNT; i++){
+		initPlayer(player);
+		players[i] = player;
+		players[i].x = 1+irandom(BOARD_WIDTH-3);
+		players[i].y = 1+irandom(BOARD_HEIGHT-3);
+		players[i].xPrevious = players[i].x;
+		players[i].yPrevious = players[i].y;
+	}
+}
+
+void updatePlayers(){
+	for (int i = 0; i < PLAYER_AMOUNT; i++){
+		if (players[i].isAlive){
+			if (i==0)
+				updateUserMovement(&players[i].horizontalSpeed, &players[i].verticalSpeed);	
+			else
+				updateBotMovement(players[i].x,players[i].y,&players[i].horizontalSpeed, &players[i].verticalSpeed);	
+				
+			players[i].x += players[i].horizontalSpeed;
+			players[i].y += players[i].verticalSpeed;
+		}
+	}
+}
+
+void playersCollision(){
+	playersCollisionWithBoard();
+	playersCollisionWithOtherPlayers();
+}
+void playerDie(Player *player){
+	if (player->isAlive){
+		playerCount--;
+		player->isAlive = false;
+	}
+}
+
+void playersCollisionWithBoard(){
+	for (int i = 0; i < PLAYER_AMOUNT; i++){
+		if (players[i].isAlive){
+			if (isColidingWithBoard(players[i].x,players[i].y)){
+				playerDie(&players[i]);
+			}
+		}
+	}
+}
+
+void playersCollisionWithOtherPlayers(){
+	for (int i = 0; i < PLAYER_AMOUNT; i++){
+		if (players[i].isAlive){
+			for (int j = 0; j < PLAYER_AMOUNT; j++){
+				if (i != j){
+					if (players[i].x == players[j].x && players[i].y == players[j].y){
+						playerDie(&players[i]);
+					}
+				}
+			}
+		}
+	}
+}
+
+void drawPlayers(){
+	for (int i = 0; i < PLAYER_AMOUNT; i++){
+			drawCharWithOffset(players[i].xPrevious, players[i].yPrevious, " ");
+
+			players[i].xPrevious = players[i].x;
+			players[i].yPrevious = players[i].y;
+
+		if (players[i].isAlive)
```

Comment:
Please, follow the codee conventions.

Reply:
I'm sorry, could you be more specific to which code covention should I follow?

Reply:
Does not make sense to implement in all code ifs with brackets and others without.

Reply:
Sure, sorry about that!